### DISCUSSION TITLE
How active is hexo in 2024?

### DISCUSSION THREAD

**Discussion by j-d-carmichael** [hexojs/hexo#5561] [General]
https://hexo.io/docs/

New to hexo. I found it after finally having enough of Wordpress... so far so good, things seem to install well, importer works ok etc.

But then when i read the comments on the docs, they are 6 years old.

Based on this it raised an obvious questions, is this project and old and dying one or still thriving?

**Comment by stevenjoezhang** [comment]
Hexo was born more than ten years ago, so you can find many old comments, which are proof of the community's heritage. Hexo is still actively updated, and many themes and plugins are also maintained by the large community.

**Comment by yoshinorin** [comment]
As @stevenjoezhang mentioned, Hexo is still actively. In the case of Hexo, each features are [separated into plugins (repositories)](https://github.com/orgs/hexojs/repositories), so the fact that there are no changes in the [hexo repository](https://github.com/hexojs/hexo) does not mean that inactive. 

However, (this is not limited to Hexo orgs) all maintainers are hobbyist, so we have to balance their time with their private lives. Therefore, there are often times when they cannot respond or investigate issues.
